src/main/java/frc/trigon/robot/utilities/FilesHandler.java

```diff
@@ -0,0 +1,99 @@
+package frc.trigon.robot.utilities;
+
+import edu.wpi.first.wpilibj.Filesystem;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FilesHandler {
+    public static final String DEPLOY_PATH = Filesystem.getDeployDirectory().getPath() + "\\";
+
+    /**
+     * Deletes the given file.
+     *
+     * @param absolutePath the file's absolute path
+     * @throws IOException if the method failed to delete the specified file
+     */
+    public static void deleteFile(String absolutePath) throws IOException {
+        File file = new File(absolutePath);
+        if(!file.delete())
+            throw new IOException("Failed to delete the file \"" + absolutePath + "\".");
+    }
+
+    /**
+     * Writes the given string to a file.
+     *
+     * @param absolutePath the file's absolute path
```

Comment:
every time you have `absolutePath` you should have the same explanation for it, in all functions